homeassistant/components/devialet/strings.json

```diff
@@ -18,5 +18,13 @@
     "abort": {
       "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
     }
+  },
+  "exceptions": {
+    "upnp_error": {
+      "message": "Unable to play media. UPnP setup not ready for Devialet device."
```

Comment:
What can the user do about this?

Reply:
The idea is, when you select a source (radio browser for example) and playback is not possible, you get noticed instead of waiting and nothing happens...

Reply:
But usually when UPnP is not available, browsing media is also not supported. This should (almost) never happen.

Reply:
Yes, but what does `UPnP setup not ready` mean for the user? What do they need to do to fix it

Reply:
There is no known solution, beside maybe restart the speaker.